andela/ah-django-unchained
# #162948850 Implement tagging functionality for articles

#### What does this PR do?
Allows users to tag articles 

#### Description of Task to be completed?
When creating an article, users should be able to add tag(s)


#### How should this be manually tested?

- Clone the repo locally
- cd ah-django-unchained
- git checkout ft-tag-articles-162948850
- get started with the app as stated in the README section
- to add an article ensure you obtain a token from logging in or signing up if not signed up
- Using the `POST api/articles/` endpoint, have the following JSON object. Note that image uploads are optional.
```
{
    "title": "My story",
    "description": "I am very happy",
    "body": "A story about me",
    "tagList": ["reactjs", "angularjs", "dragons"]

}
```
#### Screenshots

```POST ARTICLE```
![image](https://user-images.githubusercontent.com/27801956/51662869-3d10af00-1fc6-11e9-9f31-cdc888952551.png)

```UPDATE ARTICLE```
![image](https://user-images.githubusercontent.com/27801956/51663674-72b69780-1fc8-11e9-8965-3520b61c8287.png)


#### What are the relevant pivotal tracker stories?
[#162948850](https://www.pivotaltracker.com/story/show/162948850)
